File: Python/1491. Average Salary Excluding the Minimum and Maximum Salary.py
class Solution:
    def average(self, salary: List[int]) -> float:
#         salary.sort()
#         salary.pop()
#         salary.pop(0)
#         return sum(salary) / len(salary)

        return (sum(salary) - max(salary) - min(salary)) / (len(salary) - 2)

File: C/1491. Average Salary Excluding the Minimum and Maximum Salary.c
double average(int* salary, int salarySize){
    int temp = 0, max = 0, min = *salary;

    for (int i = 0; i < salarySize; i++) {
        temp += *(salary + i);
        if (*(salary + i) < min) min = *(salary + i);
        if (*(salary + i) > max) max = *(salary + i);
    }

    return (temp - max - min) / (salarySize - 2.0);
}
